locust/shape.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+
+class LoadTestShape(object):
+    """
+    A simple load test shape class used to control the shape of load generated
+    during a load test.
+    """
+
+    start_time = time.monotonic()
+
+    def reset_time(self):
+        """
+        Resets start time back to 0
+        """
+        self.start_time = time.monotonic()
+
+    def get_run_time(self):
+        """
+        Calculates run time in seconds of the load test
+        """
+        return round(time.monotonic() - self.start_time)
```

Comment:
I'm not sure a rounded time is a good thing to expose. This can give people the illusion that tick is called every second (which cannot be guaranteed because of gevent)

Reply:
OK